department-of-veterans-affairs/abd-vro
# End 2 end test improvements

## What was the problem?


Since the only dev collection (350) gave errors during exam ordering end to end test did not succeed.

Associated tickets or Slack threads:

- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How does this fix it?[^1]


I added a mock service to  mas. This currently always return success for exam ordering based on what prodtest MAS returned on Friday. This also give an opportunity to have a end to end test case for all test cases which has not been done yet.

## How to test this PR
- Build application with ENV=end3end-test
- Run end2endTesty


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
